apps/web/app/(app)/onboarding/Steps.tsx

```
@@ -0,0 +1,95 @@
+  const router = useRouter();
+  const stepRefs = useRef<(HTMLLIElement | null)[]>([]);
+
+  useEffect(() => {
+    const stepIndex = selectedStep - 1;
+    if (stepIndex >= 0 && stepIndex < steps.length) {
+      stepRefs.current[stepIndex]?.scrollIntoView({ behavior: "smooth" });
+    }
+  }, [selectedStep, steps.length]);
+
+  return (
+    <ul role="list" className="space-y-6">
```

Biome

[error] Using the role attribute 'list' on the 'ul' element is redundant, because it is implied by the semantic 'ul' element.

Unsafe fix: Remove the role attribute.


(lint/a11y/noRedundantRoles)